src/main/java/me/mrCookieSlime/Slimefun/Events/AndroidMineEvent.java

```diff
@@ -0,0 +1,41 @@
+package me.mrCookieSlime.Slimefun.Events;
+
+import org.bukkit.block.Block;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+public class AndroidMineEvent extends Event implements Cancellable {
+
+    private static final HandlerList handlers = new HandlerList();
+
+    private final Block block;
+    private boolean cancelled;
+
+    public AndroidMineEvent(Block block) {
+        this.block = block;
+    }
+
+    public static HandlerList getHandlerList() {
+        return handlers;
+    }
+
+    public HandlerList getHandlers() {
+        return handlers;
+    }
+
+    public Block getBlock() {
```

Comment:
Is this the block broken or the android? Could we get the android if not?

You should add javadocs to this since it's an API class.